### PR TITLE
Skin maths

### PR DESCRIPTION
**Pull request by mikeyrayvon**

### CODE REVIEW

File: cordova/www/games/math/math.js [hunk 1]
```diff
@@ -1,13 +1,19 @@
...
+  buttonVal: null,
   targetNumber: null,
   $targetNumber: $('#target-number'),
   input: null,
   $mathForm: $('#math-form'),
   introDialog: [
       "Lets not just exercise those leg muscles, lets get it on with the brain muscle too.",
-      "Time to do some badboy/badgyal maths. Here is the challenge yeah",
-      "I'm going to show you a number and you have to write the most complicated math you can to get to that number",
+      "Time to do some MATTHHHH!!...",
```
Comment (varffs) [inline cordova/www/games/math/math.js]: this sucks as a copy change. Me and cas were saying that as we make different games by different people we should mainly write our own copy so the voices of the game are a bit different and interesting

File: cordova/www/games/math/index.html [hunk 2]
```diff
@@ -38,23 +38,34 @@
...
-          </p>
-
-          <form id="math-form">
-            <h3>Target is: <span id="target-number"></span></h3>
-
-            <input type="text" placeholder="DO MATHS HERE" />
-
-            <button type="submit">Mark me mark me</button>
-          </form>
+          <p id="math-target">Target is: <span id="target-number"></span></p>
```
Comment (varffs) [inline cordova/www/games/math/index.html]: I think a `<p>` tag here is wrong. Should be a div.

**Comment by varffs** [comment]
There is quite a serious UI bug with the display on the calculator, if you enter too many characters you the overflow is hidden so you cant see anything you are adding. I don't really want to merge something with a bug even though there is a hurry, so a solution for this is needed. I'm not going to fix this today/right now.

**Comment by mikeyrayvon** [comment]
so should i fix that?

**Comment by varffs** [comment]
Yes please

**Comment by mikeyrayvon** [comment]
i think i have it fixed @varffs can u peep?

**Pull request closed without merging**